content/docs/contributing.md

```diff
@@ -8,29 +8,37 @@ Participation in Open Source projects is highly encouraged, and the Open Source
 
 ### Summary
 
-- Upstream contributions are highly encouraged
+We highly encourage contributions to existing open source projects. There are some requirements before you start patching, and guides to help you figure out the right path:
+
+- Patches generally don't require any review from the Open Source Program Office
 - Double-check you are not sharing confidential information
-- Avoid contributions with information or code that gives us an advantage over competitors
-- Only sign whitelisted CLAs, reach out to the Open Source team for advice
+- Please associate your commit with your saucelabs.com email unless you have a history of contributing to the repo under a different email before your employment at Sauce Lavs. [How to associate your commit with a saucelabs.com email.](https://help.github.com/articles/setting-your-email-in-git/)
```

Comment:
Why? What is the reasoning behind this?

Reply:
This is more an advise than a strict requirement. However if someone from Sauce makes an upstream patch, this patch should be associated with a Sauce Labs email since it is made out of someones working hours.

Reply:
This would be too tricky to control, should people commit with a different email during the weekends?
I am fine if the wording is changed to declare that this is a suggestion.

Reply:
I sent out an email to our legal team to get more information on that. By Californian law for example everything an engineer in our HQ puts on GitHub is owned by Sauce Labs, no matter if it is on the weekend or during the week. This is different for other countries which is why I requested support from legal.

Reply:
I think this should be added _after_ we hear from Legal about this. I still don't understand what is the point of making this a requirement. We could make the requirement that the GitHub user belongs to a Sauce Labs managed GitHub org.
(Also, there is a typo on the Sauce Labs name)

Reply:
The typo is fixed.

I tried to soften the wording to make it not sound like a requirement. However we should strongly encourage employees to have their commits signed with a Sauce email when they do work for Sauce Labs. This is just standard procedure in many companies.